### ISSUE TITLE
T - Thank you section shown after sending browser compatibility report should be properly hidden from screen readers, not just visually hidden

### ISSUE THREAD

**Issue by Neurrone** [mdn/kuma#7309]
**Summary**

The section shown (the h2 that says "thank you for your report") just after the browser compatibility table should not be visible screen readers when a report has not been sent but still is. It is only visually hidden, not completely hidden.

**Steps To Reproduce (STR)**
How can we reproduce the problem?_

1. With the NVDA screen reader and Firefox, visit any page that has the browser compatibility table such as https://developer.mozilla.org/en-US/docs/Web/HTTP/CSP
2. With NVDA, navigate to just after the browser compatibility table

**Actual behavior**

This section appears, starting with the h2 that thanks you for your report

**Expected behavior**

This should not appear at all

**Additional context**

**Comment by atopal** [comment]
Timeboxed to 1pt, will decide what to do, if it's much bigger.

**Comment by peterbe** [comment]
If it helps, the DOM element can quickly be found with `document.querySelectorAll('.bc-signal-block h2')`

**Comment by Neurrone** [comment]
When will these changes be visible in the live MDN site?

Was just curious as I still see these "hidden" elements.

**Comment by schalkneethling** [comment]
> When will these changes be visible in the live MDN site?
> 
> Was just curious as I still see these "hidden" elements.

This has been reviewed and merged. I suspect we will have a production push today(/cc @escattone 😄 ) after which this will be live.
